spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::StorageManager::CloudVolume do
+
+  let(:ems_cloud) { FactoryBot.create(:ems_ibm_cloud_power_virtual_servers_cloud) }
+  let(:ems_storage) { FactoryBot.create(:ems_ibm_cloud_power_virtual_servers_storage, :parent_ems_id => ems_cloud.id) }
+  let(:cloud_volume) { FactoryBot.create(:cloud_volume_ibm_cloud_power_virtual_servers, :ext_management_system => ems_storage ) }
+
+  describe "#validate_delete_volume" do
+  context "status is in-use" do
+    let(:status) { "in-use" }
+
+    it "validates unavailable" do
+      def validate_delete_volume
```

Comment:
Looks like this is just a copy of [`validate_delete_volume`](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb#L31) from our `CloudVolume` class.

I think there's still some fundamental misunderstanding here. We don't want to copy these class methods to the test, we want to call them from the test.

Reply:
Yeah @rspradeep you were a lot closer with this: https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/b9ecdf97efa719e9b2a5916dbe9cebd8fb5cc5ca/spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume_spec.rb
In that you were testing the methods on the objects rather than copying the method into the tests.

What would happen if we modified the method and introduced a bug but didn't update the tests?  The tests would still be passing,